requirements-transitive.txt

```diff
@@ -11,6 +11,7 @@ gevent==21.12.0
 greenlet==1.1.1
 idna==2.10
 imagesize==1.2.0
+importlib-metadata==4.13.0
```

Comment:
CI failure for flake8 check: https://github.com/reddit/experiments.py/actions/runs/3324937039/jobs/5497213395

fix: https://stackoverflow.com/a/73932581/4260179